warehouse/Dockerfile.local

```diff
@@ -0,0 +1,26 @@
+FROM python:3.9-buster
+
+LABEL org.opencontainers.image.source https://github.com/cal-itp/data-infra
+
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
+RUN apt-get update \
+  && apt-get install -y nodejs
```

Comment:
We've noticed installing `build-essentials` is needed for Python images and M1 users.

See e.g. cal-itp/benefits#795 and cal-itp/eligibility-server#85

Reply:
Nice, thanks! Adding.